### ISSUE TITLE
Missing role from aria-level definition

### ISSUE THREAD

**Issue by LJWatson** [w3c/aria#641]
The definition for [aria-level](https://w3.org/tr/wai-aria-1.1#aria-level) states:
">This can be applied inside trees to tree items, to headings inside a document, to nested grids, nested tablists and to other structural items that may appear inside a container or participate in an ownership hierarchy...
This attribute is applied to elements that act as leaf nodes within the orientation of the set, for example, on elements with role treeitem rather than elements with role group.">

The treeitem role is not included in the list of "used in" roles though.

**Comment by joanmarie** [comment]
@LJWatson: It is already included in the "inherits into" roles in the next row of that table (`treeitem` inherits `aria-level` as a supported property from its superclass role `listitem`). Thus I believe that the problem as described in the summary is not a bug.

Having said that, if this separation of roles into "used in" and "inherits into" groups is confusing or not particularly helpful, then we should consider modifying the tools that generate the table to improve the presentation.

If you could confirm my conclusion on the former, and provide your thoughts on the latter, it would be great. Thanks!

**Comment by jnurthen** [comment]
Closing due to no response
